app/questionnaire/questionnaire_schema.py

```diff
@@ -1385,22 +1385,25 @@ def get_item_anchor(self, section_id: str, list_name: str) -> str | None:
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
 
-    def _update_placeholder_dependencies(
-        self, placeholder_answer_ids: set, block: ImmutableDict
+    def _update_dependencies_for_placeholders(
+        self, answer_ids: set[str], block: ImmutableDict
```

Comment:
Does this need to be `set` as per style guide make it asbtract, i.e Iterable[str]`?

Reply:
Updated 👍